### PR TITLE
Fix Kubeless test

### PR DESCRIPTION
**Pull request by rakesh-garimella**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [ ] I have updated the relevant documentation.
---

**Fix kubeless tests based on changes to API**

Changes proposed in this pull request:

- API is now is aware of the hostname that is being used to expose lambdas. Update the kubeless tests to cover this scenario.
- Now lambdas are part of configmap and are mounted as volume. This should make them aware of the cluster they are in (eg. If they are running in minikube or they are running on AKS)

### CODE REVIEW

File: resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-kubeless-config
```
Comment (sayanh) [inline resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml]: Let's use helm templating, `test-{{ template "fullname" . }}`

File: resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  namespace: kyma-system
```
Comment (sayanh) [inline resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml]: `namespace: kyma-system` is redundant, lets leave it to helm as we do in other resources used for testing

File: resources/core/charts/kubeless/templates/tests/test-kubeless.yaml [hunk 3]
```diff
@@ -78,9 +78,23 @@ spec:
...
     - name: KUBELESS_NAMESPACE
       value: kyma-system
     - name: KUBELESS_CONFIG
       value: {{ template "fullname" . }}-config
+    - name: DOMAIN_NAME
+      value: {{ .Values.global.domainName }}
+  volumes:
+    - name: k8syaml
+      configMap: 
+        name: test-kubeless-config
```
Comment (sayanh) [inline resources/core/charts/kubeless/templates/tests/test-kubeless.yaml]: helm templating is required for the name of the ConfigMap

File: resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml (hunk 1 above)
Comment (rakesh-garimella) [reply resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml]: Done.

File: resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml (hunk 2 above)
Comment (rakesh-garimella) [reply resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml]: Done.

File: resources/core/charts/kubeless/templates/tests/test-kubeless.yaml (hunk 3 above)
Comment (rakesh-garimella) [reply resources/core/charts/kubeless/templates/tests/test-kubeless.yaml]: Done.

**Pull request merged**